LadybugTools_Engine/Python/src/ladybugtools_toolkit/plot/_seasonality.py

```diff
@@ -0,0 +1,128 @@
+"""Method to plot the seasonality of an EPW file."""
+
+import matplotlib.dates as mdates
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import pandas as pd
+from ladybug.epw import EPW
+from matplotlib.collections import PatchCollection
+
+from ..ladybug_extension.epw import (seasonality_from_day_length,
+                                     seasonality_from_month,
+                                     seasonality_from_temperature)
+from .utilities import contrasting_color
+
+
+def seasonality_comparison(epw: EPW, ax: plt.Axes = None, **kwargs) -> plt.Axes:
+    """_"""
+
```

Comment:
Kwargs are only used for season colours for the plot - compared to other plot methods, usually there is a kwarg for the axis title and other similar styling things.
Kwargs are also usually passed to the mpl plot method, though I agree it's not possible in this method specifically.

Could you add a docstring for this?